klippy/util.py

```diff
@@ -125,6 +125,17 @@ def get_cpu_info():
     model_name = dict(lines).get("model name", "?")
     return "%d core %s" % (core_count, model_name)
 
+def get_device_info():
+    try:
```

Comment:
a `with open('/proc/device-tree/model', 'r'):` should do the same , and be py2.7 compatible, and eliminate the `f.close()`call

Reply:
Yeah, and there are 2 others above :D
Technically, there are 2 examples of `f = open()` vs  1 `with open() as f`, what an omission